webcompat/form.py

```diff
@@ -380,6 +382,18 @@ def domain_name(url):
     return domain
 
 
+def switch_problem_subtype(type):
```

Comment:
[no unit tests](https://github.com/webcompat/webcompat.com/blob/f3cce74a422c51d492ef9a1772108001ad5aa105/tests/unit/test_form.py#L4) have been created for this function.

Reply:
This function will be removed (as per this [conversation](https://www.screencast.com/t/QG8mxGSg)), that's why there is no need to create a unit test.